pcsx2/gui/AppConfig.cpp

```diff
@@ -810,26 +810,26 @@ void AppConfig::FilenameOptions::LoadSave( IniInterface& ini )
 // ------------------------------------------------------------------------
 AppConfig::GSWindowOptions::GSWindowOptions()
 {
-	CloseOnEsc				= true;
-	DefaultToFullscreen		= false;
-	AlwaysHideMouse			= false;
-	DisableResizeBorders	= false;
-	DisableScreenSaver		= true;
-
-	AspectRatio				= AspectRatio_4_3;
-	Zoom					= 100;
-	StretchY				= 100;
-	OffsetX					= 0;
-	OffsetY					= 0;
-
-	WindowSize				= wxSize( 640, 480 );
-	WindowPos				= wxDefaultPosition;
-	IsMaximized				= false;
-	IsFullscreen			= false;
-	EnableVsyncWindowFlag	= false;
+	CloseOnEsc                      = true;
+	DefaultToFullscreen             = false;
+	AlwaysHideMouse                 = false;
+	DisableResizeBorders            = false;
+	DisableScreenSaver              = true;
+
+	AspectRatio                     = AspectRatio_4_3;
+	FMVAspectRatioSwitch            = FMV_AspectRatio_Switch_Off;
+	Zoom                            = 100;
+	StretchY                        = 100;
+	OffsetX                         = 0;
+	OffsetY                         = 0;
+
+	WindowSize                      = wxSize( 640, 480 );
+	WindowPos                       = wxDefaultPosition;
+	IsMaximized                     = false;
+	IsFullscreen                    = false;
+	EnableVsyncWindowFlag           = false;
```

Comment:
IMO in most cases it's not a good idea to align things.

It may look pretty but I think it makes it harder to read the code because when you read from left to right (which is what actually matters in most cases) there's now a large whitespace gap that you have to skip to read the next part of the line. This:
```
WindowSize = wxSize( 640, 480 );
```
is IMO easier to read than
```
WindowSize                                                                              = wxSize( 640, 480 );
```
Also, it unnecessarily increases the diff, Anyone making future changes might have to adjust a lots of lines when they otherwise might only want to add/adjust/remove one line because it'll ruin the existing formatting. And the more lines that are modified, the greater the likelihood of a merge conflict if multiple people are working on nearby areas of code. It also makes it harder for a reviewer to see what's really changed (I don't really want to use the ignore whitespace option all the time).

Reply:
Yeah those are good points, I'll make some adjustments.